draft-ietf-tls-esni.md

```diff
@@ -1175,6 +1188,23 @@ as authoritative for the public name, this may trigger the retry logic described
 in {{rejected-ech}} or result in a connection failure. A proxy which is not
 authoritative for the public name cannot forge a signal to disable ECH.
 
+## Deployment Impact {#no-sni}
+
+Some use cases which depend on information ECH encrypts may break with the
+deployment of ECH. The extent of breakage depends on a number of external
+factors, including, for example, whether ECH can be disabled, whether or not
+the party disabling ECH is trusted to do so, and whether or not client
+implementations will fall back to TLS without ECH in the event of such a
+toggle.
+
+Depending on implementation details and deployment settings, use cases
+which depend on plaintext TLS information may require fundamentally different
+approaches to continue working. For example, in managed enterprise settings,
+one approach may be to disable ECH entirely and for client implementations
+to honor this request. Another approach may be to intercept and decrypt client
```

Comment:
It might be worth suggesting how to disable ECH here. (Toggle in DNS?)

Reply:
We could offer that as an example, yeah.

Reply:
I don't think we should point people to unauthenticated DNS toggles if we can help it. Perhaps just "disable ECH entirely via group policy"?

Reply:
That's a good suggestion -- I'll work it in.